package.json

```diff
@@ -37,9 +37,9 @@
   "homepage": "https://github.com/unshiftio/readystate",
   "browser": "./browser.js",
   "devDependencies": {
-    "assume": "~2.0.0",
-    "istanbul": "~0.4.5",
-    "mocha": "~3.5.0",
-    "pre-commit": "~1.2.0"
+    "assume": "2.1.0",
+    "mocha": "5.2.0",
+    "nyc": "13.2.0",
```

Comment:
@binarykitchen can we revert this? It breaks npm scripts. Or update them to work with nyc :)

Reply:
It's mostlikely mocha that is missing up as the latest version is no longer compatible with older version of node which uses ES5 instead of ES6 features.

Reply:
ah right, sorry. reverted back to istanbul and all npm scripts work again now.